[it] Traduzione Cloud Native Apps (#1093)
cncf/glossary: content/it/cloud_native_apps.md

---

> title: Cloud Native Apps

Reviewer: ```suggestion
title: Applicazioni Cloud Native
```

status: Completed
category: Concetto
---

## Cos'è

> Le applicazioni Cloud Native sono progettate specificamente per sfruttare le innovazioni nel [Cloud Computing](it/cloud_computing/). Queste applicazioni si integrano facilmente con le rispettive architetture cloud sfruttandone le risorse e le capacità in termini di [scalabilità](it/scalability/). La definizione si riferisce anche ad applicazioni che sfruttano le innovazioni in ambito infrastrutturale, introdotte dal Cloud Computing. Con applicazioni Cloud Native oggi si intendono app che vengono eseguite indifferentemente su data center di un provider cloud o su piattaforme Cloud Native on-premise.

Reviewer: ```suggestion
Le applicazioni Cloud Native sono progettate specificamente per sfruttare le innovazioni nel [Cloud Computing](it/cloud_computing/). Queste applicazioni si integrano facilmente con le rispettive architetture cloud sfruttandone le risorse e le capacità in termini di [scalabilità](it/scalability/). La definizione si riferisce anche ad applicazioni che sfruttano le innovazioni in ambito infrastrutturale, introdotte dal Cloud Computing. Con applicazioni Cloud Native oggi si intendono app che vengono eseguite indifferentemente in data center di un provider cloud o su piattaforme Cloud Native on-premise.
```
con datacenter di solito si intende un luogo fisico in cui sono installati dei server.

Author: ecco, non so mai come gestirmi con le preposizioni 😅

> ## Quali problematiche risolve

Reviewer 2: ```suggestion
## Quali problematiche affronta
```

> I tradizionali ambienti on-premise fornivano risorse di calcolo in modo abbastanza personalizzato. Ogni data center disponeva di servizi basati su [architetture_con_un forte_accoppiamento](/tightly_coupled_architectures/) tra applicazioni e ambiente specifico, con un pesante provisioning manuale dell'infrastruttura, come [macchine virtuali](it/virtual_machine/) e servizi. Questo, a sua volta, ha vincolato gli sviluppatori e le loro applicazioni a quel data center specifico. Ma le applicazioni non progettate per il cloud non potevano sfruttarne le capacità di resilienza e scalabilità (pensiamo, per esempio, alle app che richiedendo un intervento manuale per avviarsi correttamente non possono essere ridimensionate automaticamente, né possono essere riavviate automaticamente in caso di errore).

Reviewer 2: ```suggestion
I tradizionali ambienti on-premise fornivano risorse di calcolo in modo abbastanza personalizzato. Ogni data center disponeva di servizi basati su [architetture_con_un forte_accoppiamento](/it/tightly_coupled_architectures/) tra applicazioni e ambiente specifico, con un pesante provisioning manuale dell'infrastruttura, come [macchine virtuali](/it/virtual_machine/) e servizi. Questo, a sua volta, ha vincolato gli sviluppatori e le loro applicazioni a quel data center specifico. Ma le applicazioni non progettate per il cloud non potevano sfruttarne le capacità di resilienza e scalabilità (pensiamo, per esempio, alle app che richiedendo un intervento manuale per avviarsi correttamente non possono essere ridimensionate automaticamente, né possono essere riavviate automaticamente in caso di errore).
```

Author: @fsbaraglia quindi /it/ anche se il termine non è ancora stato localizzato? 👍🏻 alles klar

> I tradizionali ambienti on-premise fornivano risorse di calcolo in modo abbastanza personalizzato. Ogni data center disponeva di servizi basati su [architetture_con_un forte_accoppiamento](/it/tightly_coupled_architectures/) tra applicazioni e ambiente specifico, con un pesante provisioning manuale dell'infrastruttura, come [macchine virtuali](/it/virtual_machine/) e servizi. Questo, a sua volta, ha vincolato gli sviluppatori e le loro applicazioni a quel data center specifico. Ma le applicazioni non progettate per il cloud non potevano sfruttarne le capacità di resilienza e scalabilità (pensiamo, per esempio, alle app che richiedendo un intervento manuale per avviarsi correttamente non possono essere ridimensionate automaticamente, né possono essere riavviate automaticamente in caso di errore).

Reviewer: ```suggestion
I tradizionali ambienti on-premise fornivano risorse di calcolo in modo abbastanza specifico. Ogni data center disponeva di servizi caratterizzati dal [forte accoppiamento](/it/tightly_coupled_architectures/) tra applicazioni e il loro ambiente di esecuzione, spesso dipendendo marcatamente dal provisioning manuale dell'infrastruttura, come [macchine virtuali](/it/virtual_machine/) e servizi. Questo, a sua volta, vincolava gli sviluppatori e le loro applicazioni a quel data center specifico. Ma le applicazioni non progettate per il cloud non potevano sfruttarne le capacità di resilienza e scalabilità; si pensi, per esempio, alle app che richiedendo un intervento manuale per avviarsi correttamente: esse non possono essere ridimensionate automaticamente, né possono essere riavviate automaticamente in caso di errore.
```
Ho tolto le parentesi perché mi sembrava un periodo un po' troppo lungo

Author: quanto sei bravo @sistella

## In che modo aiuta

> Sebbene non esista un modello universale per le applicazioni Cloud Native, è possibile identificare alcuni elementi comuni. Le app Cloud Native sono resilienti, gestibili e supportate dalla suite di servizi cloud che le accompagna, servizi che solitamente consentono un elevato grado di [osservabilità](/observability/) che permette agli utenti di rilevare e risolvere i problemi prima che questi si intensifichino. In combinazione con una solida automazione, consentono agli ingegneri di apportare modifiche ad alto impatto frequentemente e in modo prevedibile, con il minimo sforzo.

Reviewer 2: ```suggestion
Sebbene non esista un modello universale per le applicazioni Cloud Native, è possibile identificare alcuni elementi comuni. Le app Cloud Native sono resilienti, gestibili e supportate dalla suite di servizi cloud che le accompagna, servizi che solitamente consentono un elevato grado di [osservabilità](/it/observability/) che permette agli utenti di rilevare e risolvere i problemi prima che questi si intensifichino. In combinazione con una solida automazione, consentono agli ingegneri di apportare modifiche ad alto impatto frequentemente e in modo prevedibile, con il minimo sforzo.
```

> Sebbene non esista un modello universale per le applicazioni Cloud Native, è possibile identificare alcuni elementi comuni. Le app Cloud Native sono resilienti, gestibili e supportate dalla suite di servizi cloud che le accompagna, servizi che solitamente consentono un elevato grado di [osservabilità](/it/observability/) che permette agli utenti di rilevare e risolvere i problemi prima che questi si intensifichino. In combinazione con una solida automazione, consentono agli ingegneri di apportare modifiche ad alto impatto frequentemente e in modo prevedibile, con il minimo sforzo.

Reviewer: Tutto per il tentativo di tradurre anche "suite" :D: per me è un concetto chiaramente legato al bundle di servizi/applicazioni che costituisce un'offerta composita..
```suggestion
Sebbene non esista un modello universale per le applicazioni Cloud Native, è possibile identificare alcuni elementi comuni. Le app Cloud Native sono resilienti, gestibili e supportate dall'offerta di servizi cloud che le accompagna. Quest'ultima consente un elevato grado di [osservabilità](/it/observability/) che permette agli utenti di rilevare e risolvere i problemi prima che questi si intensifichino. In combinazione con una solida automazione, le applicazioni Cloud Native consentono agli ingegneri di apportare modifiche ad alto impatto frequentemente e in modo prevedibile, con il minimo sforzo.
```
